tests/test_forms.py

```diff
@@ -41,13 +41,25 @@ def test_valid_data(self):
         """undocumented"""
         User = get_user_model()
         user = User.objects.create_user(email="normaluser@user.com", first_name="testfirst", last_name="testlast", password="testpassword")
+<<<<<<< HEAD
         data = {
             'email': user.email,
             'first_name': 'testfirst',
             'last_name': 'testlast',
             'password': 'testpassword',
         }
         form = CustomUserChangeForm(data, instance=user)
+=======
+        form = CustomUserChangeForm(
+            {
+                'email': 'normaluser@user.com',
+                'first_name': 'testfirst',
+                'last_name': 'testlast',
+                # 'password': 'testpassword',
+            }
+            , instance=user
+        )
+>>>>>>> 8fe81fdaf00d27045757d66368fb90aa9ee936e7
```

Comment:
@enpuyou You’ll have to go through and manually remove the merge markers.

Reply:
Just removed it! Forgot to save files when resolving the merge conflicts. The newest commit should pass.